pkg/aggregate/aggregationParams.go

```diff
@@ -44,8 +48,7 @@ func (as *AggregationParams) CanAggregate(partitionAggr AggrType) bool {
 	// if interval and rollup are not even divisors we need higher resolution (3x) to smooth the graph
 	// when we add linear/spline graph projection we can reduce back to 1x
 	return ((aggrMask & partitionAggr) == aggrMask) &&
-		as.Interval >= as.rollupTime &&
-		(as.Interval%as.rollupTime == 0 || as.Interval/as.rollupTime > 3) &&
+		(as.Interval/as.rollupTime > as.getServerAggregateCoefficient() || (as.Interval == as.rollupTime && as.disableClientAggregation)) &&
```

Comment:
If `Interval` is `rollupTime`*2, we don't want to use server aggregations anymore? Also not sure I understand what `disableClientAggregation` is for.